### ISSUE TITLE
Tooltips/hints in gTile window

### ISSUE THREAD

**Issue by albertvaka** [gTile/gTile#59]
There are two toggle-state buttons at the bottom left of the gTile window (the main window, with the grid, that pops up when clicking the tray icon) that I have no idea what they do. 

Showing a tooltip when hovering them with the mouse would make them more discoverable.

**Comment by scherepanov** [comment]
Do you have any references how to make tooltips in gnome-shell?

**Comment by albertvaka** [comment]
It seems there is nothing in gnome-shell that you can use. You can take a look at this implementation, though:

https://github.com/andyholmes/gnome-shell-extension-gsconnect/blob/master/src/shell/tooltip.js

It is used like this:

https://github.com/andyholmes/gnome-shell-extension-gsconnect/blob/master/src/shell/device.js#L47

**Comment by scherepanov** [comment]
Can you submit PR with code changes for tooltip?

**Comment by paulodeleo** [comment]
@albertvaka (and anyone else looking for this information) by looking at the names of the files `animation-icon.png` and `auto-close-icon.png` and after some experimentation, I got that the first icon (the "AAA") toggles the animation of the changes preview. The second icon (looks like an... on/off power button?) toggles the auto-closing of the gTile window after applying the changes.

For now I'm unable to help adding the tooltips, but for now I opened https://github.com/gTile/gTile/pull/61 to at least mention it on readme.
